invenio_communities/communities/services/components.py

```diff
@@ -300,6 +300,20 @@ def update(self, identity, data=None, record=None, errors=None, **kwargs):
                 record["theme"] = data["theme"]
 
 
+class CommunityParentComponent(ServiceComponent):
+    """Service Component for Community parent."""
+
+    def update_parent(self, identity, data=None, record=None, **kwargs):
+        """Update parent community of a community."""
+        try:
+            parent = self.service.record_cls.pid.resolve(data)
+            if parent.parent:
+                raise ValidationError("Child community cannot be parent.")
```

Comment:
```suggestion
                raise ValidationError("Assigned parent community cannot also have a parent.")
```

IIUC, this is to prevent having 2-levels of parents?

Reply:
Yes